.github/workflows/build.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       - name: "run go build"
         run: go build -v ./...
 
+      - name: "Staticcheck"
```

Comment:
use https://github.com/marketplace/actions/run-golangci-lint better?

golangci-lint is a fast Go linters aggregator that runs multiple linters and integrates the functionality of the previous Go Meta Linter. I understand it is currently the recommended tool.

Reply:
Somehow, nothing shows for golangci-lint. see https://github.com/smiletrl/incubator-seata-go/pull/1, https://github.com/smiletrl/incubator-seata-go/actions/runs/7697184584/job/20973667110?pr=1

Reply:
<img width="901" alt="Screenshot 2024-01-29 at 22 02 23" src="https://github.com/apache/incubator-seata-go/assets/2764799/c126ca79-467b-47e1-ae30-2a3050388568">
@wt-better  ^